southpark-search/tests/distributed/terraform.tf

```diff
@@ -139,7 +32,33 @@ resource "aws_ecs_task_definition" "southpark_task" {
         }
       ],
       "memory": 512,
-      "cpu": 256
+      "cpu": 10
+    },
+    {
+      "name": "encoder",
+      "image": "416454113568.dkr.ecr.us-east-2.amazonaws.com/sp-repo",
+      "essential": true,
+      "portMappings": [
+        {
+          "containerPort": 80,
+          "hostPort": 80
+        }
+      ],
+      "memory": 512,
+      "cpu": 10
+    },
+    {
+      "name": "indexer",
+      "image": "416454113568.dkr.ecr.us-east-2.amazonaws.com/sp-repo",
+      "essential": true,
+      "portMappings": [
+        {
+          "containerPort": 8080,
```

Comment:
Furthermore, you have to give the right ports for the `pod` access. If I get https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#container_definition_portmappings correctly, you need to specify each port by hand and port ranges are not supported by fargate. This means, you have to set each and every container communication port in the pods. This should all happen via environment variables.